tests/test_freqtradebot.py

```diff
@@ -4317,7 +4317,7 @@ def test_sync_wallet_dry_run(mocker, default_conf_usdt, ticker_usdt, fee, limit_
 @pytest.mark.usefixtures("init_persistence")
 @pytest.mark.parametrize("is_short,buy_calls,sell_calls", [
     (False, 1, 2),
-    (True, 2, 1),
+    (True, 1, 2),
```

Comment:
> Instead of having all these conditionals, I would add these values inside @pytest.mark.parametrize, you can find lots of other tests somewhere in freqtrade for an example

Like here's one for example